apps/admin_api/priv/spec.yaml

```diff
@@ -1683,7 +1685,8 @@ components:
             example:
               name: "Account Name"
               description: "The account description"
-              master: true
+              master: false
```

Comment:
What happens if we put true here? 🤔

Reply:
Oh good question. Actually the master account is determined by the system now, there will always be a master account and the rest are non-master. This attribute is not valid anymore (for create/update, but is still available for get/all). Will remove.

Reply:
Yep, that's what I was thinking about, but was not sure :)